packages/components/src/side_scroller/controls.module.scss

```diff
@@ -0,0 +1,43 @@
+@import "nebenan-ui-kit";
+
+.control {
+  // rgba(255, 255, 255, 0) is important to fade through white is Safari
```

Comment:
This comment doesn't require any action, but good to memorise this kind of things for potential future cross-browser bugs.